ymer/src/main/java/com/avanza/ymer/PersistedInstanceIdCalculationService.java

```diff
@@ -141,6 +162,9 @@ private void calculatePersistedInstanceId(String collectionName, Set<Integer> nu
 		);
 		DocumentCollection collection = spaceMirror.getDocumentDb().getCollection(collectionName);
 
+		PersistedInstanceIdStatistics statistics = jobStatistics.computeIfAbsent(collectionName, x -> new PersistedInstanceIdStatistics());
```

Comment:
When does the map here not already contain a key for `collectionName`? Is it for the case where `calculatePersistedInstanceId` is manually called for an uninitialized collection? In that case, does that mean that such a collection would not be present as a jmx mbean?

Reply:
Yes, your right that it would not be present as a jmx mbean. I don't think this should be a usual case. I mainly wrote this so that `statistics` wouldn't be set to `null`, requiring null-checks in the calculation job.

Reply:
ok!